var/spack/repos/jcsda-emc-bundles/packages/ufs-weather-model-env/package.py

```diff
@@ -12,32 +12,51 @@ class UfsWeatherModelEnv(BundlePackage):
     homepage = "https://github.com/ufs-community/ufs-weather-model"
     git = "https://github.com/ufs-community/ufs-weather-model.git"
 
-    maintainers = ["kgerheiser", "climbfuji"]
+    maintainers = ["AlexanderRichert-NOAA", "climbfuji"]
 
-    version("1.0.0")
+    version("develop")
 
     variant(
         "debug",
         default=False,
         description="Build a debug version of certain dependencies (ESMF, MAPL)",
     )
     variant("python", default=True, description="Build Python dependencies")
+    variant("shared", default=True, description="Build dynamic libraries when possible")
 
-    depends_on("base-env", type="run")
     depends_on("ufs-pyenv", type="run", when="+python")
 
-    depends_on("fms@2022.01", type="run")
-    depends_on("bacio", type="run")
-    depends_on("crtm@2.4.0", type="run")
-    depends_on("g2", type="run")
-    depends_on("g2tmpl", type="run")
-    depends_on("ip", type="run")
-    depends_on("sp", type="run")
-    depends_on("w3nco", type="run")
-
-    depends_on("esmf~debug", type="run", when="~debug")
-    depends_on("esmf+debug", type="run", when="+debug")
-    depends_on("mapl~debug", type="run", when="~debug")
-    depends_on("mapl+debug", type="run", when="+debug")
-
-    # There is no need for install() since there is no code.
+    depends_on("bacio@2.4.1")
```

Comment:
We've tried to not encode versions in these bundle packages and leave that up to the common packages.yaml and the templates. I am not sure if changing this for the ufs-weather-model-env only but keeping the rest as-is is what we should do. Is there a reason why the versions/variants can't be in configs/templates/ufs-weather-model/spack.yaml?

I am open to revisit how/where we specify versions and variants, but we should be consistent across the various templates, bundle packages, etc.